frontend/templates/category-modal.tmpl

```diff
@@ -0,0 +1,89 @@
+<form>
+    <div class="modal-header"><h3>
+        {{#if isNew}}
+            {{t "category modal.new category"}}
+        {{else}}
+            {{t "category modal.edit"}} {{model.name}}
+        {{/if}}
+    </h3></div>
+
+    <div class="modal-body">
+        <label>
+          <span class="required">{{t "category modal.name"}}</span>
+          <input id="name" type="text" value="{{model.name}}" required>
+        </label>
+
+        <label>{{t "category modal.color"}} <input type="color" value="{{model.settings.color}}"></label>
+
+        <fieldset>
+            <legend>{{t "category modal.affiliation"}}</legend>
+            <label class="radio">
+                <input type="radio" name="affiliation" value="event">
+                {{t "category modal.belongs to video"}}
+            </label>
+            <label class="radio">
+                <input type="radio" name="affiliation" value="series">
+                {{t "category modal.belongs to series"}}
+            </label>
+
+            <span class="help-block" id="no-series-affiliation-with-scale">
+                {{t "category modal.no series affiliation with scale"}}
+            </span>
+        </fieldset>
```

Comment:
The "Belong to video or series"-radio button are currently always displayed. I personally think it would make more sense to allow users to only affiliate a category with a series when that event actually belongs to a series. 

Furthermore, this seems to break things when:
1. On an event that does not belong to a series, create a public category that belongs to a series.
2. Create some annotations with the category.
3. Change the event so that it belongs to a series.
4. The created annotations cannot be fetched anymore, and new ones cannot be created using this category.

Reply:
I agree with the first part, but integration such logic would require some more work we don't have time for, currently. :disappointed: 

Similarly, either one of Lars' outstanding PRs will fix the bug you describe, OR we will accept it for now, until someone orders us to fix it. :sweat_smile: